### PR TITLE
Preparatory changes for renaming default log files

### PR DESCRIPTION
**Pull request by lucafavatella**
Extracted from https://github.com/aeternity/aeternity/pull/2049

### CODE REVIEW

**Comment by lucafavatella** [comment]
System smoke test job failed on [CI](https://circleci.com/gh/aeternity/aeternity/599):
```
%%% aest_sync_SUITE ==> net_split_recovery: FAILED
%%% aest_sync_SUITE ==> {{net1_node2,{error,socket_closed_remotely}},
 [{aest_nodes,get_block,2,
              [{file,"/home/circleci/aeternity/_build/system_test+test/extras/system_test/helpers/aest_nodes.erl"},
               {line,369}]},
  {aest_sync_SUITE,net_split_recovery,1,
                   [{file,"/home/circleci/aeternity/_build/system_test+test/extras/system_test/aest_sync_SUITE.erl"},
                    {line,512}]},
  {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1539}]},
  {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1055}]},
  {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,987}]}]}
```

I see this [tracked in Minerva](https://www.pivotaltracker.com/story/show/162814429), it is probably intermittent test failure present in master branch too.

I took the liberty of restarting CI.

**Comment by lucafavatella** [comment]
I rebased on recent master.

File: apps/aecore/src/aecore.app.src [hunk 1]
```diff
@@ -70,7 +70,6 @@
          {'$setup_hooks',
           [
            {normal, [
-                     {110, {aec_peers, check_env, []}},
```
[the remark this answers is not in the document]
Comment (lucafavatella) [reply apps/aecore/src/aecore.app.src]: I believe I did https://github.com/aeternity/aeternity/pull/2050/files#diff-60612869cf8dfc6cb271f2310f183122L157

[the remark this answers is not in the document]
Comment (zp-sd) [reply apps/aecore/src/aecore.app.src]: Yes, I missed that somehow (not sure how though, since I went though the PR twice 😛 ).

**Pull request merged**